examples/spatial/malaria_prevalence_PyMCv5.myst.md

```diff
@@ -0,0 +1,391 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: pymc
+  language: python
+  name: python3
+---
+
+(The prevalence of malaria in the Gambia)=
+# The prevalence of malaria in the Gambia
+
+:::{post} Aug 04, 2024 
+:tags: spatial, autoregressive, count data
+:category: beginner, tutorial
+:author: Jonathan Dekermanjian, bwengals (please add your name here)
```

Comment:
```suggestion
:author: Jonathan Dekermanjian, Bill Engels
```

Reply:
Oh no need to add me as an author. It's great work and I wouldn't mind being associated with it, but I didn't write a single word!

Reply:
We are collaborating on this example. Even though you did not write a single word your feedback and ideas are being incorporated into the work. I won't force your name to be added but I do think credit should be given to all contributors of a piece of work.